src/components/steps/WalletSeedBackup.vue

```diff
@@ -35,6 +35,9 @@ export default {
     ...mapState({
       seed: state => state.wallet.mnemonics,
     }),
+    allowCopyMnemonics () {
+      return process.env.NODE_ENV === 'development'
+    }
```

Comment:
Can we make this computed property and the CSS class consistent in their naming so that we don't need to negate one?